packages/server/src/automations/steps/serverLog.ts

```diff
@@ -61,8 +61,10 @@ export async function run({
   inputs: ServerLogStepInputs
   appId: string
 }): Promise<ServerLogStepOutputs> {
+  if (typeof inputs.text === "object") {
+    inputs.text = JSON.stringify(inputs.text)
+  }
   const message = `App ${appId} - ${inputs.text}`
-  console.log(message)
```

Comment:
I think this `console.log` is still required, this is the core function of the `serverLog.ts`

Reply:
So stupid on my behalf, my lint threw a warning and I wasn't even paying attention and just removed it. Sorted

Reply:
I can 100% see myself doing the same thing. It would be worth adding a comment above the line disabling the lint and explaining that this is a load bearing console.log.